frontend/packages/ux-editor/src/components/Properties/ConditionalRendering.module.css

```diff
@@ -11,7 +11,3 @@
   gap: 1rem;
   margin-bottom: 1rem;
```

Comment:
I know its not from this PR, but think we should clean up on the go - can you replace with tokens? 😄